tests/pgautofailover_utils.py

```diff
@@ -1113,6 +1113,7 @@ def destroy(self):
                 "drop",
                 "node",
                 "--destroy",
+                "--force",
```

Comment:
Do we really want to change the way we run all of our test suite to use `--force`?